external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -7,6 +7,7 @@
     -->
     <NugetRuntimeIdentifier>win7-x64</NugetRuntimeIdentifier>
     <OutputPath>$(RuntimePath)</OutputPath>
+    <OutputPath Condition="'$(BinPlaceNETCoreAppTestDependencies)' == 'true'">$(NETCoreAppPackageRuntimePath)\..\dependencies</OutputPath>
```

Comment:
While this might redirect the dependencies that come from the test-runtime folder there are still various dependencies that get copied by the test projects themselves as well which will be missed. 

Beyond that any reason to not use the bin-placing logic for this project as well, to make it additive?